attributes/default.rb

```diff
@@ -395,9 +395,12 @@
 )
 # Lustre repo string is built following the official doc
 # https://docs.aws.amazon.com/fsx/latest/LustreGuide/install-lustre-client.html
+# 'centos' is used for arm and 'el' for x86_64
+fsx_arm_prefix = arm_instance? ? 'centos' : 'el'
```

Comment:
We would want to make clear that this variable is only used for centos. To keep in line with rest of the file, how about something like `default['cfncluster']['lustre']['centos']['base_url_prefix']= arm_instance? ? 'centos' : 'el'`

We might also not need this variable depending on your approach to address the comment about the url below

Reply:
Ok I changed this to the suggest variable